circle.yml

```diff
@@ -25,16 +25,20 @@ dependencies:
     - DEBIAN_FRONTEND=noninteractive sudo apt-get autoremove -y --purge
         libblas-dev
         libbz2-dev
+        libexpat1-dev
```

Comment:
Does doing this remove `python-dev`?

https://github.com/Homebrew/brew/releases/tag/1.1.12 changes some behavior around when the `default_formula` is installed, so what we want to do about `python-dev` and `libexpat1-dev` might change after we merge 1.1.12 and later.

I'm not too familiar with the changes; see https://github.com/Homebrew/brew/pull/2389